railway_manager_interface/openapi.yaml

```diff
@@ -73,3 +106,227 @@ components:
           description: Error message describing what went wrong
       required:
         - detail
+    TowedRollingStock:
+      type: string
+      enum:
+      - AFAMOD
+      - AFAMOD2
+      - MA
+      - MAL100
+      - MALOUR
+      - MAVIDE
+      - ME100
+      - ME120
+      - ME140
+      - ME200SER
+      - MEPA
+      - TRAINLOC
+      title: TowedRollingStock

```

Comment:
I think this is supposed to be a simple string if I'm not mistaken

Towed Rolling Stocks are loaded in database the same way we load rolling stocks

Reply:
This has been fixed!